lib/TPP/LinalgDeGeneralize.cpp

```diff
@@ -152,6 +156,56 @@ struct FillOpDeGeneralizationPattern
   }
 };
 
+// From linalg.generic to TPP brgemm (VNNI).
+struct TppBrgemmDeGeneralizationPattern
+    : public OpRewritePattern<linalg::GenericOp> {
+  using OpRewritePattern<linalg::GenericOp>::OpRewritePattern;
+
+  bool isTppVnniOp(linalg::GenericOp linalgOp) const {
```

Comment:
This looks like it could be in a more generic place?

Reply:
I moved in TppUtils.h. However I think we should pull out all the `isXXXOps` from TppUtils as we are going to use them also when converting linalg to xsmm, and including `TppUtils.h` looks borken.